terraform/shared/scripts/create_elastic_users_identity.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+"""
+This script is run by manually after to create roles/users in
```

Comment:
```suggestion
This script is run manually after "terraform apply" to create roles/users in
```

It'd be good to explain why this is different to the previous script.